### PR TITLE
ta: partially revert 4a353de

### PR DESCRIPTION
**Pull request by Tim-Anderson**
Vendor variable are expected to be delete-able on current production customers.

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]

**Review by igoropaniuk — APPROVED** [review]
LGTM

File: fiovb/ta/entry.c [hunk 1]
```diff
@@ -336,11 +336,10 @@ static TEE_Result delete_persist_value(uint32_t pt,
 	name_buf_sz = params[0].memref.size;
 
 	/*
-	 * Vendor variables and rollback_protection should not be
+	 * rollback_protection should not be
```
Comment (ricardosalveti) [inline fiovb/ta/entry.c:339]: nit, mind having this comment in one single line now that is shorter?

Comment (Tim-Anderson) [reply fiovb/ta/entry.c]: done

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**